fairscale/nn/data_parallel/fully_sharded_data_parallel.py

```diff
@@ -55,6 +56,11 @@
 if TYPE_CHECKING:
     from collections import OrderedDict  # noqa: F401
 
+if os.getenv("ENABLE_NCCL_BASE_COLLECTIVES", "1") == "0":
```

Comment:
Lets add a TODO here with an open GitHub issue to track us figuring this out.

Reply:
done.